[2022-07-27 23:18:17,315] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: tutorial-python-op.start manual__2022-07-27T14:18:16.417249+00:00 [queued]>
[2022-07-27 23:18:17,326] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: tutorial-python-op.start manual__2022-07-27T14:18:16.417249+00:00 [queued]>
[2022-07-27 23:18:17,326] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-07-27 23:18:17,327] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2022-07-27 23:18:17,327] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-07-27 23:18:17,347] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): start> on 2022-07-27 14:18:16.417249+00:00
[2022-07-27 23:18:17,355] {standard_task_runner.py:52} INFO - Started process 48143 to run task
[2022-07-27 23:18:17,364] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'tutorial-python-op', 'start', 'manual__2022-07-27T14:18:16.417249+00:00', '--job-id', '64', '--raw', '--subdir', 'DAGS_FOLDER/test5.py', '--cfg-path', '/var/folders/6v/f1q42tlx39bf4xb4nltqccj00000gn/T/tmp0mn4ef9i', '--error-file', '/var/folders/6v/f1q42tlx39bf4xb4nltqccj00000gn/T/tmp6n9zico3']
[2022-07-27 23:18:17,366] {standard_task_runner.py:80} INFO - Job 64: Subtask start
[2022-07-27 23:18:17,466] {task_command.py:369} INFO - Running <TaskInstance: tutorial-python-op.start manual__2022-07-27T14:18:16.417249+00:00 [running]> on host janghuisuui-MacBookAir.local
[2022-07-27 23:18:17,583] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=your-email@g.com
AIRFLOW_CTX_DAG_OWNER=owner-name
AIRFLOW_CTX_DAG_ID=tutorial-python-op
AIRFLOW_CTX_TASK_ID=start
AIRFLOW_CTX_EXECUTION_DATE=2022-07-27T14:18:16.417249+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-07-27T14:18:16.417249+00:00
[2022-07-27 23:18:17,585] {subprocess.py:62} INFO - Tmp dir root location: 
 /var/folders/6v/f1q42tlx39bf4xb4nltqccj00000gn/T
[2022-07-27 23:18:17,586] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo "start!!!!!"']
[2022-07-27 23:18:17,604] {subprocess.py:85} INFO - Output:
[2022-07-27 23:18:17,627] {subprocess.py:92} INFO - start!!!!!
[2022-07-27 23:18:17,628] {subprocess.py:96} INFO - Command exited with return code 0
[2022-07-27 23:18:17,702] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=tutorial-python-op, task_id=start, execution_date=20220727T141816, start_date=20220727T141817, end_date=20220727T141817
[2022-07-27 23:18:17,755] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-27 23:18:17,879] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
